bodo/pandas/_executor.cpp

```diff
@@ -0,0 +1,70 @@
+#include "_executor.h"
+#include <arrow/python/api.h>
+#include <arrow/python/pyarrow.h>
+#include <arrow/status.h>
+#include <memory>
+#include "../io/arrow_compat.h"
+#include "../libs/_bodo_to_arrow.h"
+#include "_plan.h"
+#include "arrow/io/api.h"
+#include "parquet/arrow/reader.h"
+
+Executor::Executor(std::unique_ptr<duckdb::LogicalOperator> plan) {
+    // Convert logical plan to physical plan and create query pipelines
+
+    // TODO: support nodes other than read parquet
+    duckdb::LogicalGet& get_plan = plan->Cast<duckdb::LogicalGet>();
+    std::string path =
+        get_plan.bind_data->Cast<BodoParquetScanFunctionData>().path;
+    std::shared_ptr<PhysicalReadParquet> read_parquet =
+        std::make_shared<PhysicalReadParquet>(path);
+
+    pipelines.emplace_back(
+        std::vector<std::shared_ptr<PhysicalOperator>>({read_parquet}));
+}
+
+std::pair<int64_t, PyObject*> Executor::execute() {
+    // TODO: support multiple pipelines
+    return pipelines[0].execute();
+}
+
+std::pair<int64_t, PyObject*> Pipeline::execute() {
+    // TODO: support multiple operators
+    return operators[0]->execute();
+}
+
+std::pair<int64_t, PyObject*> PhysicalReadParquet::execute() {
```

Comment:
Docstring.  What are the int64_t and PyObject and you're doing a pointer cast to int64_t?  Why?  I'm sure there's a good reason but please document.

Reply:
Added comments to the header files. Our convention is to only add docstrings to header files (Google guidelines or something?).